scripts/tools/zap/zap_execution.py

```diff
@@ -0,0 +1,113 @@
+# Copyright (c) 2022 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import re
+import subprocess
+import sys
+from typing import Tuple
+
+# The version MUST be of the form `YYYY.MM.YY'
+# Since this is ordered as such, alphabetical sorting will be used to check
+# validity
+#
+# This version MUST be changed together with `${ZAP_VERSION}` in
+#    - integrations/docker/images/chip-build/Dockerfile
```

Comment:
Then we need to:

1. Clearly point from there to here (right now it's not clear over there what needs to be changed here).
2. Ideally have the same strings here and there (and post-process them here as needed) so all the changes can be done in a single `git sed` or so.

Reply:
Created a script that can do the update, added a comment here saying one should use the script.

sed is also possible but arguably uglier than the script because it would not find min version in zap_execution (but also that may not matter as much)